client/web/compose/src/components/PageBlocks/Navigation/NavTypes/Url.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <tr>
+    <td />
+    <td>
+      <b-form-group :label="$t('navigation.fieldLabel')">
+        <b-form-input
+          v-model="value.options.itemOption.label"
+          type="text"
+          size="sm"
+        />
+      </b-form-group>
+    </td>
+    <td>
+      <b-form-group :label="$t('navigation.url')">
+        <b-form-input
+          v-model="value.options.itemOption.url"
+          type="text"
+          size="sm"
+        />
+      </b-form-group>
+    </td>
+    <td>
+      <b-form-group :label="$t('navigation.openIn')">
+        <b-form-select
+          v-model="value.options.itemOption.targetType"
+          :options="openInType"
+          size="sm"
+        >
+          <template #first>
+            <b-form-select-option :value="undefined">
+              {{ $t("navigation.none") }}
+            </b-form-select-option>
+          </template>
+        </b-form-select>
+      </b-form-group>
+    </td>
+  </tr>
+</template>
+
+<script>
+export default {
+  i18nOptions: {
+    namespaces: 'block',
+  },
+
+  props: {
+    value: {
+      type: Object,
+      required: true,
+    },
+  },
+
+  data () {
+    return {
+      openInType: [
+        { value: '_self', text: this.$t('navigation.sameTab') },
+        { value: '_blank', text: this.$t('navigation.newTab') },
+        { value: '_blank', text: this.$t('navigation.newWindow') },
+      ],
+    }
+  },
+
+  watch: {
+    column: {
+      immediate: true,
+      handler (column) {
+        this.value = column
+      },
+    },
+    value (value) {
```

Comment:
Why this watcher? You should add the event to the inputs

Reply:
I made use this update the value to the parent, instead of adding change methods to the input

Reply:
If you abandon the value, and just use column and don't model but emit update instead then it should work the same with less code, same for other types